### DISCUSSION TITLE
Caracteres no titulo da notificação do Email.

### DISCUSSION THREAD

**Discussion by tecinfotecnologia** [RamonSilva20/mapos#1893] [Q&A] [answered]
Bom dia Pessoal,

Estou enfrentando o seguinte falha na notificação do e-mail esta apresentado dessa formar o cabeçalho "Ordem de ServiÃ§o".

php.ini  > default_charset = "UTF-8"

config/email.php >  $config['charset']          = 'utf-8';

Att:
Thales Almeida

**Comment by RamonSilva20** [comment, marked as answer]
Olá @tecinfotecnologia, está usando qual e-mail para envio? 
Tive um problema similar utilizando o Outlook. Para resolver precisei modificar o CRLF na biblioteca de envio.

Se for o seu caso, tente adicionar no arquivo application/libraries/MY_Email.php a propriedade public $crlf = "\r\n";


```
<?php defined('BASEPATH') or exit('No direct script access allowed');

class MY_Email extends CI_Email
{

    public $crlf = "\r\n";
    
    // DB table
    private $table_email_queue = 'email_queue';

```

**Reply by tecinfotecnologia** [reply]
Estou utilizado Gmail com relay sendinblue
Sucesso com esse paramento.

**Comment by tecinfotecnologia** [comment]
Resolvido removi o relay do gmail e configurei a conta conforme este topico #1848 .

Obrigado @RamonSilva20 e @jsfachini .
